content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -34,10 +34,18 @@ Dashboard also provides information on the state of Kubernetes resources in your
 
 ## Deploying the Dashboard UI
 
+{{< note >}}
+Kubernetes Dashboard supports only Helm-based installation currently as it is faster
+and gives us better control over all dependencies required by Dashboard to run.
+{{< /note >}}
+
 The Dashboard UI is not deployed by default. To deploy it, run the following command:
 
 ```
```

Comment:
```suggestion
```shell
```

Reply:
I forgot to mention it previously

Please change the code snippet to the shell, as it differentiates the comments from commands

Reply:
Shell output looks like this:
![screenshot](https://github.com/kubernetes/website/assets/50987400/b6d60776-d9f1-4736-9c68-bdc33cf8c0a3)

Reply:
Yep, good catch! I've updated.